db/schema.rb

```diff
@@ -164,7 +158,7 @@
     t.string   "description"
     t.datetime "start_time"
     t.datetime "end_time"
-    t.integer  "duration"
+    t.decimal  "duration",    precision: 7, scale: 2
```

Comment:
@marshall-alex unknown extra whitespace here.

Reply:
This was also unintentional